src/librustc_codegen_llvm/context.rs

```diff
@@ -247,8 +252,8 @@ impl<'ll, 'tcx> CodegenCx<'ll, 'tcx> {
         // this effect) by marking very little as `dllimport` and praying the
         // linker will take care of everything. Fixing this problem will likely
         // require adding a few attributes to Rust itself (feature gated at the
-        // start) and then strongly recommending static linkage on MSVC!
-        let use_dll_storage_attrs = tcx.sess.target.target.options.is_like_msvc;
+        // start) and then strongly recommending static linkage on Windows!
```

Comment:
Large parts of the comment above are outdated.
E.g. we don't set dllexport LLVM attributes now and export everything with `/DEF`.
It would be great to remove or update the outdated parts.

Reply:
I don't know details about how rustc handles that and I'm not confident in rewriting it 🤷‍♂️

Reply:
Ok.